api/v1alpha1/api.go

```diff
@@ -339,6 +339,11 @@ type AIServiceBackendSpec struct {
 	// +optional
 	BackendSecurityPolicyRef *gwapiv1.LocalObjectReference `json:"backendSecurityPolicyRef,omitempty"`
 
+	// Timeouts defines the timeouts that can be configured for an HTTP request.
```

Comment:
can you add a comment about default 60s ?

Reply:
Sure, the default value is only for default route, for others is nil.

Reply:
ah ok sorry nvm